git/objects/commit.py

```diff
@@ -335,8 +336,70 @@ def stats(self) -> Stats:
         return Stats._list_from_string(self.repo, text)
 
     @property
-    def trailers(self) -> Dict:
-        """Get the trailers of the message as dictionary
+    def trailers(self) -> Dict[str, str]:
+        """Get the trailers of the message as a dictionary
+
+        Git messages can contain trailer information that are similar to RFC 822
+        e-mail headers (see: https://git-scm.com/docs/git-interpret-trailers).
+
+        WARNING: This function only returns the latest instance of each trailer key
```

Comment:
I think it's OK to deprecate it, and suggest using `trailers_dict()`. Then this warning can be removed as it will show up when using it.

Reply:
Agreed - deprecated [78424b5](https://github.com/gitpython-developers/GitPython/pull/1576/commits/78424b56654ad476da4bd2faf88d3875c5265e0d)

Reply:
That's not deprecation 😅, that's downright removal, which is a breaking change.

After taking a look I realize now that there is no built-in facility for marking properties or functions as deprecated. So I think documenting the deprecation is as good as it gets.

Sorry for the hassle, but once that is reverted, it's ready to merge.

Reply:
No worries! `trailers` method is back - [9ef07a7](https://github.com/gitpython-developers/GitPython/pull/1576/commits/9ef07a731caf39684891bce5cc92c4e91829138d)